### PR TITLE
Configure environment variables in JRuby again, instead of the selfrun scripts

### PR DESCRIPTION
**Pull request by dmikurube**
@muga Can you have a look? We're dropping back the complicated part of the selfrun batch file!

A release plan of mine then is:
* This (#779) + #790 => v0.8.33
* JRuby 9.1.3.0 (#791) => v0.8.34

What do you think of the plan?

### CODE REVIEW

**Comment by dmikurube** [comment]
Delaying this after v0.8.32...

File: embulk-core/src/main/java/org/embulk/jruby/JRubyScriptingModule.java [hunk 1]
```diff
@@ -250,14 +260,13 @@ public UnrecognizedJRubyLoadPathException(final String message, final Throwable
...
         {
             if (directory != null) {
-                /* Environment variables are set in the selfrun script or bin/embulk:
-                 *   ENV['EMBULK_BUNDLE_PATH']: set through '-b' | '--bundle', or inherit from the runtime environment
-                 *   ENV['BUNDLE_GEMFILE']: set for "ENV['EMBULK_BUNDLE_PATH']/Gemfile"
-                 *   ENV['GEM_HOME']: unset
-                 *   ENV['GEM_PATH']: unset
-                 */
+                jruby.put("__internal_bundler_plugin_source_directory__", directory);
+                jruby.runScriptlet("ENV['EMBULK_BUNDLE_PATH'] = __internal_bundler_plugin_source_directory__");
```
Comment (dmikurube) [inline embulk-core/src/main/java/org/embulk/jruby/JRubyScriptingModule.java]: Ah, this may not be required.

**Comment by muga** [comment]
Your release plan sounds good to me. I'm taking a look.

**Comment by dmikurube** [comment]
Thanks. Merging!

**Pull request merged**